### ISSUE TITLE
lua 正常调用函数会报一个错误

### ISSUE THREAD

**Issue by isghost** [cocos2d/cocos2d-x#15387]
lua_cocos2dx_auto.cpp  文件里的`lua_cocos2dx_GLProgramState_setUniformFloat`函数,第一个参数即可以是整数也可以是字符串，但如果第一个参数字符串，当程序运行到

```
 ok &= luaval_to_int32(tolua_S, 2,(int *)&arg0, "cc.GLProgramState:setUniformFloat");
```

会报一个错误，break，然后运行是字符串情况下的代码。这是正常的调用函数过程。如果游戏中每帧都调用`setUniformFloat`，打印出来的log将变的非常的恶心。而且没啥问题报一个error也不太合理。类似的函数挺多的。

**Comment by minggo** [comment]
Yep, you are right. We should remove the log in `luaval_to_int32()`. And we should enhance the binding codes to make it more smart to check errors.

**Comment by feijing566** [comment]
@minggo any plant about this?

**Comment by minggo** [comment]
@feijing566 sorry, we have other more important issues to fix, so there is not plan this issue right now. You are appreciated if you can send a PR for it.
